### PR TITLE
Feature: prompt for payment type

### PR DESCRIPTION
**Pull request by 1yam**
Problem:

We currently blocking user from creating Confidential vm using hold method. However, in the case user use NFT he need to set the payment type to Hold.

### CODE REVIEW

**Comment by hoh** [comment]
Can you ensure that tests cover the changes ?

File: src/aleph_client/commands/instance/__init__.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ async def create(
         Path("~/.ssh/id_rsa.pub").expanduser(),
         help="Path to a public ssh key to be added to the instance.",
     ),
-    print_messages: bool = typer.Option(False),
+    print_messages: bool = typer.Option(True),
```
Comment (Psycojoker) [inline src/aleph_client/commands/instance/__init__.py:81]: That looks like an unrelated change to this PR

File: src/aleph_client/commands/instance/__init__.py [hunk 2]
```diff
@@ -284,19 +295,17 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
         item_hash: ItemHash = message.item_hash
 
         console = Console()
-        if crn and (confidential or not hold):
+        if crn and not hold:
             if not crn.url:
                 return
-            async with AuthenticatedAlephHttpClient(account=account, api_server=sdk_settings.API_HOST) as client:
```
Comment (Psycojoker) [inline src/aleph_client/commands/instance/__init__.py:290]: Why did you removed this first switch statement? It doesn't useful but I'm curious.

File: src/aleph_client/commands/instance/__init__.py [hunk 3]
```diff
@@ -284,19 +295,17 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
...
-                        print(status, result)
-                        echo(f"Could not start instance  {item_hash} on CRN")
-                        return
+            account = _load_account(private_key, private_key_file)
+            async with VmClient(account, crn.url) as crn_client:
+                status, result = await crn_client.start_instance(vm_id=item_hash)
+                logger.debug(status, result)
+                if int(status) != 200:
+                    print(status, result)
+                    echo(f"Could not start instance  {item_hash} on CRN")
```
Comment (Psycojoker) [inline src/aleph_client/commands/instance/__init__.py:307]:
```suggestion
                    echo(f"Could not start instance {item_hash} on CRN")
```

File: src/aleph_client/commands/instance/__init__.py (hunk 2 above)
Comment (1yam) [reply src/aleph_client/commands/instance/__init__.py:290]: I did remove it cause it was here but no used

**Comment by 1yam** [comment]
> Can you ensure that tests cover the changes ?

We currently don't have any tests for instance creation in the aleph-client. It would be interesting to create some, but how should we approach this? Aleph-client is just a CLI for the SDK, so how do we test it?

And maybe need another PR for that

**Review by philogicae — APPROVED** [review]
